modules/client/src/internal/client/decoding.ts

```diff
@@ -27,7 +27,8 @@ import {
   IpfsError,
   resolveIpfsCid,
 } from "@aragon/sdk-common";
-import { erc20ContractAbi } from "../abi/erc20";
+import { abi as ERC20_ABI } from "@openzeppelin/contracts/build/contracts/ERC20Votes.json";
```

Comment:
Shouldn't it be just the standard ERC20 contract ABI? ERC20 votes is an extension of it

Reply:
mmm, no because in some places we need to deploy the votes version, I can split the function in deploy-erc20 and deploy-erc20-votes